app/styles/themes/_high-contrast.scss

```diff
@@ -0,0 +1,73 @@
+body.theme-high-contrast {
+  // A small set of colors used to flatten ux and improve contrast
+  --hc-background: #090c11;
+  --hc-secondary-background: #323946;
+  --hc-border: #7a828e;
+  --hc-text: #f0f3f6;
+  --hc-active-item: #9ea7b3;
+  --hc-secondary-active-item: #596069;
+  --hc-active-text: #090c11;
```

Comment:
Absolutely fine with punting this to another PR but these should all be suffixed with `--color` to match the convention of the other variables.

Reply:
done